{{cookiecutter.repostory_name}}/bin/prepare-os.sh

```diff
@@ -35,3 +48,14 @@ fi
 if [ ! -x "${DOCKER_COMPOSE_BIN}" ]; then
     apt-get -y install docker-ce docker-compose
 fi
+
+if [ ! -x "${AWS_CLI}" ]; then
+    curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
```

Comment:
please add support for ARM and consistency verification. We will soon need to experiment with various kinds of hardware to measure efficiency of a 3rd party computation on various (aws) platforms, not being able to run it on ARM would be a disadvantage.